server/src/main/java/org/apache/iotdb/db/mpp/common/header/HeaderConstant.java

```diff
@@ -200,4 +209,19 @@ public class HeaderConstant {
                 new ColumnHeader(COLUMN_FUNCTION_CLASS, TSDataType.TEXT)),
             true);
   }
+
+  static {
+    showRegionHeader =
+        new DatasetHeader(
+            Arrays.asList(
+                new ColumnHeader(COLUMN_REGION_ID, TSDataType.INT32),
+                new ColumnHeader(COLUMN_TYPE, TSDataType.TEXT),
+                new ColumnHeader(COLUMN_STATUS, TSDataType.TEXT),
+                new ColumnHeader(COLUMN_STORAGE_GROUP, TSDataType.TEXT),
+                new ColumnHeader(COLUMN_SLOTS, TSDataType.INT32),
```

Comment:
CLOUMN_SLOTS's datatype should be TSDataType.INT64?

Reply:
fixed~